app/ts/PreferencesTab/Preferences/PreferencesWeights/PreferencesWeightsTable/PreferencesWeightsTable.tsx

```diff
@@ -52,10 +52,14 @@ export default function PreferencesWeightsTable() {
       (criterion: ICriterion): JSX.Element => {
         const unit = criterion.dataSources[0].unitOfMeasurement;
         const usePercentage = showPercentages && canBePercentage(unit.type);
+        const description = criterion.description ? criterion.description : '';
         return (
           <TableRow key={criterion.id}>
-            <TableCell>{criterion.title}</TableCell>
-            <TableCell>{criterion.description}</TableCell>
+            <TableCell>
+              <Tooltip disableHoverListener={!description} title={description}>
+                <span style={{color: '#0000ee'}}>{criterion.title}</span>
+              </Tooltip>
```

Comment:
extract component? If there is no description should it state so instead of having no tooltip, because it might otherwise be confusing why the criterion is colored.